streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -939,7 +939,7 @@ private StreamThread createAndAddStreamThread(final long cacheSizePerThread, fin
         return streamThread;
     }
 
-    //For KafkaStreamsTest
+    //Ensure Mockito stub construct with capture argument for KafkaStreamsTest.
```

Comment:
Could you add a space between `//` and `Ensure`?

Reply:
Have changed.